.gitignore

```diff
@@ -102,4 +102,6 @@ modules.xml
 node_modules/*
 
 ### Cypress stuff ###
-cypress/screenshots*
\ No newline at end of file
+cypress/screenshots*
+
+package-lock.json
```

Comment:
Please remove this. A `package-lock.json` shouldn't be ignored. I just don't have one because of my use of yarn.